scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/core/service/impl/nullIDException.java

```diff
@@ -0,0 +1,7 @@
+package edu.asu.diging.scriptoocloud.core.service.impl;
+
+public class nullIDException extends Exception {
```

Comment:
Classes start with an uppercase letter. Eclipse should have given you a warning when you created this class.

Reply:
I did not get a red or yellow error about this, but still an oversight on my end. Refactored, and also moved to /core/exceptions.